build.bat

```diff
@@ -19,8 +19,10 @@ IF EXIST "%programfiles(x86)%\Microsoft Visual Studio\2017\Community\Common7\Too
 )
 POPD
 
+set /p version= < src\StarcounterClientFiles\version.txt
```

Comment:
Please add information in CONTRIBUTING.md what is the source of truth for version number. Looking at this PR I am not sure:

- what is the purpose of `version.txt`. Can't https://github.com/Starcounter/StarcounterClientFiles/blob/3.x_nuget_cake/build/ClientFiles.cake read the version number directly from `AssemblyInfo.cs`?
- what is the source of truth for the version number: `version.txt` or `AssemblyInfo.cs`?

Reply:
I didn't want to parse the file in the first place, but when I see the end solution, then I guess the nuget pack script can parse that file since it is beeing done anyways.